core/src/main/java/software/amazon/awssdk/retry/backoff/BackoffStrategy.java

```diff
@@ -13,18 +13,29 @@
  * permissions and limitations under the License.
  */
 
-package software.amazon.awssdk.retry.v2;
+package software.amazon.awssdk.retry.backoff;
+
+import software.amazon.awssdk.retry.RetryPolicyContext;
+import software.amazon.awssdk.retry.SdkDefaultRetryPolicies;
 
-/**
- * Super interface for {@link RetryPolicy} that defines a strategy for backing off between retries.
- */
 public interface BackoffStrategy {
 
+    BackoffStrategy DEFAULT = new FullJitterBackoffStrategy(SdkDefaultRetryPolicies.BASE_DELAY,
+                                                            SdkDefaultRetryPolicies.MAX_BACKOFF_IN_MILLIS,
+                                                            SdkDefaultRetryPolicies.DEFAULT_NUM_RETRIES);
+
+    BackoffStrategy NONE = new FixedDelayBackoffStrategy(1);
+
     /**
      * Compute the delay before the next retry request. This strategy is only consulted when there will be a next retry.
      *
      * @param context Context about the state of the last request and information about the number of requests made.
      * @return Amount of time in milliseconds to wait before the next attempt. Must be non-negative (can be zero).
      */
     long computeDelayBeforeNextRetry(RetryPolicyContext context);
```

Comment:
Would `Duration` be better than `long` here?

Reply:
Yeah that should be fine. Will update.